swatch-contracts/src/main/java/com/redhat/swatch/contract/security/RbacRolesAugmentor.java

```diff
@@ -61,7 +63,17 @@ public Uni<SecurityIdentity> augment(
         && isCustomer(rhIdentityPrincipal)) {
       return context.runBlocking(() -> lookupRbacRoles(identity));
     }
-    return Uni.createFrom().item(identity);
+    return Uni.createFrom().item(buildWithAllRoles(identity));
```

Comment:
I think this is fine for dev mode, but inappropriate otherwise. See `RolesAugmentor.augment` for the mappings that are appropriate outside of dev mode.

Reply:
Yes, this is an oversight on my side. Fixed.